pkgs/applications/networking/sniffers/wireshark/default.nix

```diff
@@ -160,10 +162,9 @@ stdenv.mkDerivation rec {
   '';
 
   postInstall = ''
-    # to remove "cycle detected in the references"
-    mkdir -p $dev/lib/wireshark
-    mv $out/lib/wireshark/cmake $dev/lib/wireshark
-  '' + (if stdenv.isDarwin && withQt then ''
+    moveToOutput $out/lib/cmake $dev
```

Comment:
I'm not sure if this is the cause but I am getting this while building:
```
wireshark-qt> checking for references to /build/ in /nix/store/656h1583nni7wslmznp3c77nx7sxcqk7-wireshark-qt-4.2.0...
wireshark-qt> gzipping man pages under /nix/store/656h1583nni7wslmznp3c77nx7sxcqk7-wireshark-qt-4.2.0/share/man/
wireshark-qt> patching script interpreter paths in /nix/store/656h1583nni7wslmznp3c77nx7sxcqk7-wireshark-qt-4.2.0
wireshark-qt> stripping (with command strip and flags -S -p) in  /nix/store/656h1583nni7wslmznp3c77nx7sxcqk7-wireshark-qt-4.2.0/lib /nix/store/656h1583nni7wslmznp3c77nx7sxcqk7-wireshark-qt-4.2.0/bin
wireshark-qt> find: '/nix/store/xhpmr4c6ha4h5kkfiagv2snpryw53hqc-wireshark-qt-4.2.0-dev': No such file or directory
wireshark-qt> postPatchMkspecs
```

Reply:
It was because the `dev` output wasn't created, although it was stated in `outputs`. Also, no development files have been installed at all. Fixed by installing the `Development` component.